### PR TITLE
Add “services” and “metadata” to NodeInfo

### PR DESCRIPTION
**Pull request by MFTabriz**
“services” and “metadata” are both required by the [NodeInfo schema version 2.0](http://nodeinfo.diaspora.software/ns/schema/2.0).

### CODE REVIEW

**Review by ClearlyClaire — APPROVED** [review]

File: app/serializers/nodeinfo/serializer.rb [hunk 1]
```diff
@@ -37,6 +37,10 @@ def open_registrations
     Setting.registrations_mode != 'none' && !Rails.configuration.x.single_user_mode
   end
 
+  def metadata
+    Array([])
```
Comment (nightpool) [inline app/serializers/nodeinfo/serializer.rb:41]: `Array([])` is not idiomatic ruby code. Please use `[]` instead, which is identical.

**Review by nightpool — CHANGES_REQUESTED** [review]

**Review by Gargron — APPROVED** [review]

File: app/serializers/nodeinfo/serializer.rb (hunk 1 above)
Comment (MFTabriz) [reply app/serializers/nodeinfo/serializer.rb]: Sorry! This was literally my first time reading/writing Ruby.

**Comment by MFTabriz** [comment]
@ClearlyClaire, @Gargron: Can you please check the requested changes by @nightpool? It's a trivial change and I addressed the issue in less than 30 min but did not get any feedback afterwards.

**Comment by MFTabriz** [comment]
@ClearlyClaire, @Gargron: Can you please check the requested changes by @nightpool? It's a trivial change and I addressed the issue in less than 30 min but did not get any feedback afterwards.

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**